src/content/app/genome-browser/state/track-config/trackConfigSlice.ts

```diff
@@ -68,52 +98,190 @@ export const updateTrackStatesAndSave: ActionCreator<
   browserStorageService.saveTrackStates(trackStates);
 };
 
+export const getTracksStateInfo = (
+  genomeId: string,
```

Comment:
What does the genomeId do in this function? Can't find where it's used. And if it isn't, how did eslint allow you to commit this?

Reply:
✅